main.py

```diff
@@ -473,12 +493,13 @@ def gcs_to_pubsub(cloud_event: CloudEvent):
             observe_gcp_content_type=content_type,
         )
 
-    # delete the file from the bucket
+     # delete the file from the bucket
     if blob.exists():
         blob.delete()
         logging.info("Blob successfully deleted")
     else:
         logging.warning("Blob not found, could not delete")
+    return "Sucessfully processed buket", 200
```

Comment:
nit: you've successfully processed the file, not a whole bucket (also bucket is misspelled)

Reply:
Successfully is also misspelled :(